benchmarker/modules/problems/resnet50/mxnet.py

```diff
@@ -2,7 +2,7 @@
 from mxnet.gluon.model_zoo import vision
 
 
-def get_kernel(params, unparsed_args):
+def get_kernel(params):
     parser = argparse.ArgumentParser(description='Benchmark resnet50')
     parser.parse_args(unparsed_args)
```

Comment:
```suggestion

```
This line should be deleted. `unparsed_args` is not defined

Reply:
need mxnet tests :)